dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/RenormalizationProcessorTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.jobs.steps.processors;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+import uk.ac.ebi.eva.dbsnpimporter.sequence.FastaSequenceReader;
+
+import java.nio.file.Paths;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+
+/**
+ * Check that the RenormalizationProcessor only modifies ambiguous variants, and changes the positions and alleles
+ * correctly otherwise.
+ *
+ * The tests use the fasta in the resources folder, which should start with "TGCGCCA".
+ */
+public class RenormalizationProcessorTest {
+
+    private static FastaSequenceReader fastaSequenceReader;
+
+    private static RenormalizationProcessor renormalizer;
+
+    @BeforeClass
+    public static void setUpClass() throws Exception {
+        fastaSequenceReader = new FastaSequenceReader(Paths.get("src/test/resources/Gallus_gallus-5.0.test.fa"));
+        renormalizer = new RenormalizationProcessor(fastaSequenceReader);
+    }
+
+    @AfterClass
+    public static void tearDownClass() throws Exception {
+        fastaSequenceReader.close();
+    }
+
+    @Test
+    public void nonAmbiguousReplacements() throws Exception {
+        assertNonAmbiguousDoesNotChange(3, "C", "T");
+        assertNonAmbiguousDoesNotChange(3, "CG", "GC");
+    }
+    @Test
+    public void nonAmbiguousInsertions() throws Exception {
+        assertNonAmbiguousDoesNotChange(3, "", "A");
+        assertNonAmbiguousDoesNotChange(3, "", "AA");
+        assertNonAmbiguousDoesNotChange(3, "", "AAA");
```

Comment:
Maybe a comment could clarify why these are not ambiguous? I assume it is because there is no ambiguity in the reference sequence, but explaining it a bit would make it more clear because we observed some cases that were exactly like these.

Reply:
Agree, I was a bit afraid of comments getting outdated, but it's true that without more context it's hard to understand